### ISSUE TITLE
Integrate noise reduction feature from Dasharo

### ISSUE THREAD

**Issue by UndeadDevel** [Nitrokey/heads#32]
**Is your feature request related to a problem? Please describe.**
As time goes on the laptop (NV41 Nitropad) is getting louder, especially when a performance governor is used for the CPU; this is on QubesOS 4.2 using NitroKey Heads 2.1. The noise is a kind of blistering or high frequency cracking sound, which, on performance governor, can be heard even in regular indoor environments (~50 dB ambient sound) and is really loud in a quiet indoor environment (~25 dB ambient). It is likely related to the issue described [here](https://github.com/dasharo/dasharo-issues/issues/224).

**Describe the solution you'd like**
In the above linked issue Wessel from NovaCustom confirms that the new Dasharo binaries apparently fix the problem. It would be nice if the next NitroKey Heads release could incorporate that fix.

**Describe alternatives you've considered**
Alternatively consider using e.g. thermal pads in certain places inside the laptop, which may also help reduce the noise, but that won't fix the issue for those who already bought the laptop.

**Additional context**
If I may add, it would also be nice if the new PGP authentication feature for the recovery shell as well as the TPM Disk Unlock Key functionality were added to NitroKey Heads from upstream linuxboot Heads.

**Comment by commandline-be** [comment]
> > Alternatively consider using e.g. thermal pads in certain places inside the laptop, which may also help reduce the noise, but that won't fix the issue for those who already bought the laptop.
> 
> * You might want to [do this yourself](https://forum.qubes-os.org/t/otherss-who-bought-a-nv41-and-have-noise-issues/20436/3).

[novacustom](https://forum.qubes-os.org/u/novacustom)
[6d](https://forum.qubes-os.org/t/otherss-who-bought-a-nv41-and-have-noise-issues/20436/6)
[@savory](https://forum.qubes-os.org/u/savory) @ anyone reading this;

Apart from the suggested improvement, [Acoustic Noise Mitigation](https://github.com/Dasharo/dasharo-issues/issues/224#issuecomment-1836397053) in the [beta firmware version](https://novacustom.com/forum/d/15-call-for-beta-testers/7) should help to reduce the interference sounds even further.


Thus far I've not noticed loud noise from the NV50 but it does heat up rapidly under some conditions. Oddily enough this is a known issue with i5-1240P CPU and Linux but not Microsoft. Hence here also likely firmware issue at hand.

**Comment by daringer** [comment]
We have a v2.4 in the making (actually already in test) using the latest 1.7.1 Dasharo Coreboot. This should help with the fan noise, apart from many other fixes (like proper suspend support). It is currently in testing, but we'll likely won't release it before Christmas, but directly during the first days in 2024.

This will not include LUKS TPM handling and also not the PGP auth feature for the recovery shell - currently there is no plan to support them, but we'll take a look how and if we can incorporate them.

**Comment by UndeadDevel** [comment]
Good to hear about 2.4 coming soon!

Unfortunately the [Acoustic Noise Mitigation fix](https://github.com/Dasharo/coreboot/commit/cad41c1bf3ded334b93c80a4dca45b462592318f) seems to be in 1.7.2, but having proper suspend support is a step in the right direction.

Merry Christmas!

**Comment by daringer** [comment]
https://github.com/Nitrokey/heads/tree/nitropad-release-v2.4 now uses 1.7.2 including noise mitigation - I expect a release in the first week of january after another full test round.

**Comment by UndeadDevel** [comment]
Confirmed fixed with latest NK Heads (2.4), thank you! Also a big thanks for enabling the PGP auth as well as TPM DUK features! Great Christmas present (for Orthodox Christmas :laughing: )!

There are two regressions, however (reboot doesn't work anymore from Heads and battery charge limits don't work anymore); I'll open new issues in a moment.

I also still get the "Card counters: PIN is not set - set PIN before the first use" warning after the "HOTP code verification application" starts and detects my NK3 (it's a line under "Connected device status:", below the line "Firmware: v4.11"), even though I have set both Admin and User PIN on the NK3A Mini after updating the NK firmware to 1.6...as I understand it's benign but it should probably be removed nonetheless.
